app/models/miq_queue.rb

```diff
@@ -31,6 +31,26 @@ class MiqQueue < ApplicationRecord
   PRIORITY_WHICH  = [:max, :high, :normal, :low, :min]
   PRIORITY_DIR    = [:higher, :lower]
 
+  def self.artemis_events_client
+    @artemis_client ||= begin
+      require "manageiq-messaging"
+      ManageIQ::Messaging.logger = _log
+      queue_settings = Settings.workers.worker_base.queue_worker_base.event_handler
```

Comment:
If these are really being used just from the `MiqQueue` class I feel like these settings are in the wrong place.

Reply:
yeah - should we introduce a top level `:miq_queue:` setting? Or stuff it under `:prototype:` for now?

Reply:
prototype for now is fine...by default we should still be using MiqQueue anyway, so that bit will have to change in this PR before merge.